### DISCUSSION TITLE
filemtime(): stat failed for /www/public/wp-content/themes/bootscore-child/assets/scss/editor.scss

### DISCUSSION THREAD

**Discussion by finalwebsites** [bootscore/bootscore#901] [Q&A]
Hi,

first I thought I get this warning only on my local workstation, but also on my test server (which is a regular setup).
How can I solve this?
The file is not missing.

**Comment by crftwrk** [comment]
Hi,

where I can see this error? What are the steps to reproduce?

**Comment by finalwebsites** [comment]
I solved it by copying the editor.scss tfile to the child theme. I'm not sure why this happens.
The warrning was visible on any page in the frontend.

**Reply by crftwrk** [reply]
That's because the `editor.scss` is enqueued to child:

https://github.com/bootscore/bootscore/blob/50c871b0198b580c3b14fa48e1053db61bca6010/inc/enqueue.php#L73

And part of child since v6.0.0. https://github.com/bootscore/bootscore-child/blob/main/assets/scss/editor.scss. Think your child is still on v6.0.0-beta1.

See https://bootscore.me/blog/bootscore-6/#editor-style-theme-json-and-block-patterns for what this file is used for.

**Reply by finalwebsites** [reply]
I see, it looks like I need to merge the changes. It's not so common that you need to update a child theme as well.
